### ISSUE TITLE
Update vcpkg-docs to describe new way of working with vcpkg included in visual studio

### ISSUE THREAD

**Issue by MatthijsPrent** [microsoft/vcpkg-docs#389]
### Type of issue

Outdated article

### Description

I believe this information page is correct and working. However, it is described that vcpkg should be manually checked out and build. Since VS 2022 17.6 the vcpkg package manager can be included as an installable component which could simplify this workflow. I think it would be good to update this description to describe how to use the installable VS componend.

https://devblogs.microsoft.com/cppblog/vcpkg-is-now-included-with-visual-studio/

### Page URL

https://learn.microsoft.com/en-us/vcpkg/get_started/get-started-msbuild

### Content source URL

https://github.com/Microsoft/vcpkg-docs/blob/main/vcpkg/get_started/get-started-msbuild.md

### Document Version Independent Id

d90305f7-2d67-3e22-6477-cfb1e21c1260

### Article author

@data-queue

### Metadata

* ID: d90305f7-2d67-3e22-6477-cfb1e21c1260 
* Service: **vcpkg**

**Comment by patrikhuber** [comment]
I agree, especially the `vcpkg integrate install` step, I think that one is even no longer recommended, as the recommended way nowadays (for the sake of reproducible builds) is to use `CMakePresets.json` and specify
```
"CMAKE_TOOLCHAIN_FILE": {
  "value": "$env{VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake",
  "type": "FILEPATH"
}
```
.
